drfc_manager/models/storage_operations.py

```
@@ -2,44 +1,44 @@
-            prefix=f"{model_name}/",
-            recursive=True
+            env_vars.DR_LOCAL_S3_BUCKET, prefix=f"{model_name}/", recursive=True
         )
         for obj in objects:
             storage_client.client.remove_object(
-                storage_client.config.bucket_name,
-                obj.object_name
+                env_vars.DR_LOCAL_S3_BUCKET, obj.object_name
             )
     except Exception as e:
         raise StorageError(f"Error deleting model {model_name}: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)